msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -603,7 +590,7 @@ public void getAccount(@NonNull final String oid,
     }
 
     private void getBrokerAndLocalAccount(@NonNull final String oid,
-                                          @NonNull final AccountLoadedCallback callback) {
+                                          @NonNull final AccountsLoadedCallback callback) {
```

Comment:
This should generate a warning for using a raw type - will this only ever return 1?